utilities/utility.py

```diff
@@ -29,6 +40,21 @@ def validate_country_field(**kwargs):
         raise AttributeError("Not a valid country")
 
 
+def validate_name_field(**kwargs):
+    """
+    Function to validate location fields when
+    saving an object
+    :params kwargs
+    """
+    my_args = []
+    locations = ['Lagos', 'Kampala', 'Nairobi']
```

Comment:
Good work @michael-basweti 
Jus a clarification I need, I was wondering since Rwanda opened recently should it also be included on the list of locations, and if there is any rationale for omitting it kindly lemme know. Cheers!

Reply:
Rwanda has not been added to Converge yet, will be added later